application.py

```diff
@@ -12,4 +12,4 @@
 app = Flask('app')
 
 create_app(app)
-application = WhiteNoise(app, STATIC_ROOT, STATIC_URL)
+app.wsgi_app = WhiteNoise(app.wsgi_app, STATIC_ROOT, STATIC_URL)
```

Comment:
It's likely fine but just keen to understand why you didn't take across the `max_age=WhiteNoise.FOREVER` param here? I'm guessing it's something to do with how we serve frontend assets but haven't given it enough thought?

Reply:
mostly that I didn't want to risk breaking things. From the admin PR:

> Since we version our asset filenames there’s no need for a browser to ever fetch the same file twice.

I'm not sure how our doc dl assets are managed (they're definitely different to admin, which no longer uses whitenoise at all but instead uploads to s3 from concourse and hosts from there), so didn't want to risk accidentally leaving old files cached forever.

Reply:
![image](https://user-images.githubusercontent.com/7228605/73959287-08397680-4901-11ea-9fa7-e7a4071d1a1b.png)

I think we are versioning our assets based on this screenshot but would tempted to get a second opinion from Chris or Tom before being confident.

Regardless, this is fine as is